### ISSUE TITLE
Add more programming language details and complete roadmap

### ISSUE THREAD

**Issue by PJijin** [PJijin/Show-Path#10]
- Add more programming language/library and complete the roadmap 
- Add programming language details. (Name, Free video tutorials (FreeCodeCamp or any other source), description, good articles to read related the programming language)

Feel free to check the [CONTRIBUTING.md](https://github.com/PJijin/Show-Path/blob/master/CONTRIBUTING.md) file  😇

Check demo on https://showpath.tech

**Comment by Iambizi** [comment]
Hi ! Looking to add new programming language details, along with the roadmap to it's frameworks, npm packages and testing frameworks but with no details yet for roadmap part, just the names for now. 

Would you except all details to be added as well or could I create a separate issue to build on it if PR is approved ? 

Thanks for your time !

**Comment by PJijin** [comment]
@Iambizi  Yes. 😃Add roadmap first and once it is approved can add the details section.

**Comment by Iambizi** [comment]
@PJijin Awesome sounds good !
